src/base/static/js/routes.js

```diff
@@ -138,7 +162,10 @@ Shareabouts.Util = Util;
       if (Backbone.history.getFragment() === '') {
         startPageConfig = S.Util.findPageConfig(options.pagesConfig, {start_page: true});
 
-        if (startPageConfig && startPageConfig.slug) {
+        if (startPageConfig 
+          && startPageConfig.slug
+          // don't route to the start page on small screens
```

Comment:
It was just a feature we added at sone point--I think the idea was that the start page on mobile obscured too much of the map and we thought it would be better for users to see as much of the map as possible when they first load the page